src/components/AddContact.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { Button, TextField, Dialog, DialogTitle, DialogContent, DialogContentText, DialogActions } from '@material-ui/core';
 import { isValidChecksum } from '@iota/checksum';
 import { contactStore } from 'src/stores/ContactStore';
+import {ActiveDialog, settingStore} from "../stores/SettingStore";
 
 
 interface IPorps {
```

Comment:
the open prop is not used anymore, remove. use setting store as well for the open state in the component. Add @observer to all Dialog components. Make this changes also to the other dialog components or even better, use one component for all dialogs.